tools/ratbagc.py.in

```diff
@@ -101,36 +101,43 @@ class RatbagDeviceType(IntEnum):
 
 class RatbagdUnavailable(Exception):
     """Signals DBus is unavailable or the ratbagd daemon is not available."""
+
     pass
 
 
 class RatbagError(Exception):
     """A common base exception to catch any ratbag exception."""
+
     pass
 
 
 class RatbagErrorDevice(RatbagError):
     """An exception corresponding to RatbagErrorCode.DEVICE."""
+
     pass
 
 
 class RatbagErrorCapability(RatbagError):
     """An exception corresponding to RatbagErrorCode.CAPABILITY."""
+
     pass
 
 
 class RatbagErrorValue(RatbagError):
     """An exception corresponding to RatbagErrorCode.VALUE."""
+
     pass
 
 
 class RatbagErrorSystem(RatbagError):
     """An exception corresponding to RatbagErrorCode.SYSTEM."""
+
     pass
 
 
 class RatbagErrorImplementation(RatbagError):
     """An exception corresponding to RatbagErrorCode.IMPLEMENTATION."""
+
     pass
```

Comment:
We can remove all these `pass` statements.

Reply:
For some reason I though this would needa higher Python version. Thanks!